### PR TITLE
Feature v0.17.0 compose scripts

### PR DESCRIPTION
**Pull request by afeiszli**

### CODE REVIEW

**Comment by afeiszli** [comment]
To test nm-quick and nm-quick-interactive, use this branch: https://github.com/gravitl/netmaker/tree/test_v0.17.0_compose (uses branch name and test image for downloads. This branch will have correct branch names / images only when in production)

File: compose/docker-compose.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ version: "3.4"
 services:
   netmaker:
     container_name: netmaker
-    image: gravitl/netmaker:v0.16.3
+    image: gravitl/netmaker:testing
```
Comment (mattkasun) [inline compose/docker-compose.yml:6]: this should be v0.17.0

File: compose/docker-compose.yml [hunk 2]
```diff
@@ -42,39 +42,27 @@ services:
...
-    labels:
-      - traefik.enable=true
-      - traefik.http.routers.netmaker-api.entrypoints=websecure
-      - traefik.http.routers.netmaker-api.rule=Host(`api.NETMAKER_BASE_DOMAIN`)
-      - traefik.http.routers.netmaker-api.service=netmaker-api
-      - traefik.http.services.netmaker-api.loadbalancer.server.port=8081
   netmaker-ui:
     container_name: netmaker-ui
-    image: gravitl/netmaker-ui:v0.16.3
+    image: gravitl/netmaker-ui:testing
```
Comment (mattkasun) [inline compose/docker-compose.yml:47]: v0.17.0

File: docker/mosquitto.conf [hunk 3]
```diff
@@ -1,7 +1,7 @@
 per_listener_settings false
 listener 8883
 protocol websockets
-allow_anonymous false
+allow_anonymous true
```
Comment (mattkasun) [inline docker/mosquitto.conf:4]: this should remain as false

**Review by mattkasun — CHANGES_REQUESTED** [review]

File: docker/mosquitto.conf (hunk 3 above)
Comment (afeiszli) [reply docker/mosquitto.conf]: resolved

**Review by mattkasun — APPROVED** [review]

**Review by 0xdcarns — APPROVED** [review]

**Pull request merged**